site/content/docs/5.0/components/offcanvas.md

```diff
@@ -0,0 +1,259 @@
+  </div>
+</div>
+{{< /example >}}
+
+## Color schemes
+
+Easily style an offcanvas element with a different `background-color` or `color` with our [color utilities]({{< docsref "/utilities/colors" >}}).
+
+{{< example >}}
+<button class="btn btn-primary" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasColored" aria-expanded="false" aria-controls="offcanvasColored">Colored offcanvas</button>
+
+<div class="offcanvas offcanvas-left bg-dark text-white" tabindex="-1" id="offcanvasColored" aria-labelledby="exampleOffCanvasLiveLabel2">
+  <div class="offcanvas-header">
+    <h5 class="modal-title" id="exampleOffCanvasLiveLabel2">Colored offcanvas</h5>
+    <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close">
+    </button>
+  </div>
+  <div class="offcanvas-body">
+    <p>
+      Bootstrap is an open source and intuitive CSS framework, which is used primarily for mobile first front-end website development.
+      It was discovered by Mark Otto and Jacob Thornton at Twitter, and it was formerly named as ‘Twitter Blueprint’.
+    </p>
+    <p>
+      CSS is principally used, but HTML templates are also a part of it.  Besides HTML and CSS templates,
+      it also depends closely on JavaScript components, mostly in the type of jQuery plugins.
+    </p>
+    <p>
+      Bootstrap also has widespread support and vast coding documentation with extensive online resources and assistance from the community of developers.
+      This, in turn, makes it easier to understand this framework better and how to employ it.
+    </p>
+  </div>
+</div>
+
+{{< /example >}}
+
+## Options
+
+By default, when an offcanvas is visible, the `<body>` of your page cannot be scrolled. You can use the following data-options to change this behavior:
+
+- `data-bs-body="scroll"` enables scrolling on the `<body>` when offcanvas is open
+
+{{< example >}}
+<button class="btn btn-primary" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasExample4" aria-expanded="false" aria-controls="offcanvasExample4">Enable body scrolling </button>
+
+<div class="offcanvas offcanvas-left" data-bs-body="scroll" tabindex="-1" id="offcanvasExample4" aria-labelledby="exampleOffCanvasLiveLabel3">
+  <div class="offcanvas-header">
+    <h5 class="modal-title" id="exampleOffCanvasLiveLabel3">Colored with scrolling</h5>
+    <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close">
+    </button>
+  </div>
+  <div class="offcanvas-body">
+    <p>Try scrolling the rest of the page to see this option in action.</p>
+  </div>
+</div>
+{{< /example >}}
+
+## Accessibility
+
+Be sure to add aria-labelledby="...", referencing the modal title, to .offcanvas. Note that you don’t need to add role="dialog" since we already add it via JavaScript.
```

Comment:
I'd start this with a short mention a la "The off-canvas panel is, conceptually, a modal dialog."